packages/hardhat-core/test/internal/hardhat-network/provider/utils/assertEqualTraces.ts

```diff
@@ -3,8 +3,8 @@ import { assert } from "chai";
 import { RpcDebugTraceOutput } from "../../../../../src/internal/hardhat-network/provider/output";
 
 export function assertEqualTraces(
-  expected: RpcDebugTraceOutput,
-  actual: RpcDebugTraceOutput
+  actual: RpcDebugTraceOutput,
```

Comment:
Pretty hard to review this, so I'm asking. Do you also updated the uses of this?

Reply:
No, because they were all wrongly passing the actual trace first, which was making the messages in failing tests very confusing :sweat_smile: Plus, I think passing the actual value first and the expected second is idiomatic (at least in chai).

Reply:
Oh, wait, there is one place where the expected trace is passed first. I'll fix that.